app/services/github_client/organization.rb

```diff
@@ -1,9 +1,14 @@
 module GithubClient
   class Organization < GithubClient::Base
     MAX_REPOSITORIES_PER_PAGE = 100
+    MAX_USERS_PER_PAGE = 30
 
     def repositories
       get_all_paginated_items('orgs/rootstrap/repos', MAX_REPOSITORIES_PER_PAGE)
     end
+
+    def members
+      get_all_members('/orgs/rootstrap/members', MAX_USERS_PER_PAGE)
```

Comment:
I'm pretty sure you can use `get_all_paginated_items` here instead of creating a new method

Reply:
Actually there is a difference in how the response is parsed in those two methods.
 `new_items = JSON.parse(response.body).map(&:with_indifferent_access)`
        vs
 `new_users = JSON.parse(response.body).map { |user| user['login'] }`